docs/modules/ROOT/pages/how-tos/cloudscale/setup-storage-cluster.adoc

```diff
@@ -0,0 +1,363 @@
+= Setup a storage cluster
+
+:delabel_app_nodes: yes
+
+[abstract]
+--
+Steps to deploy an https://products.docs.vshn.ch/products/appuio/managed/storage_cluster.html[APPUiO Managed Storage Cluster] on an OpenShift 4 cluster on https://cloudscale.ch[cloudscale.ch].
+--
+
+== Starting situation
+
+* You already have an OpenShift 4 cluster on cloudscale.ch
+* You want to deploy the APPUiO Managed Storage Cluster addon on that OpenShift 4 cluster.
+* You have admin-level access to the OpenShift 4 cluster on which you want to setup a storage cluster.
+
+== Prerequisites
+
+* `kubectl`
+* `yq` https://mikefarah.gitbook.io/yq[yq YAML processor] (version 4 or higher)
+* `commodore`, see https://syn.tools/commodore/running-commodore.html[Running Commodore]
+
+== Steps
+
+. Configure API access
++
+[source,bash]
+----
+export COMMODORE_API_URL=https://api.syn.vshn.net <1>
+export COMMODORE_API_TOKEN=<your lieutenant token>
+
+# Set Project Syn cluster and tenant ID
+export CLUSTER_ID=<lieutenant-cluster-id> # Looks like: c-cluster-id-1234
+export TENANT_ID=$(curl -sH "Authorization: Bearer ${COMMODORE_API_TOKEN}" ${COMMODORE_API_URL}/clusters/${CLUSTER_ID} | jq -r .tenant)
+----
+<1> Replace with the API URL of the desired Lieutenant instance.
+
+. Create a local directory to work in and compile the cluster catalog
++
+[source,bash]
+----
+export WORK_DIR=/path/to/work/dir
+mkdir -p "${WORK_DIR}"
+pushd "${WORK_DIR}"
+
+commodore catalog compile "${CLUSTER_ID}"
+----
++
+[TIP]
+====
+We strongly recommend creating an empty directory, unless you already have a work directory for the cluster you’re about to work on.
+This guide will run Commodore in the directory created in this step.
+====
+
+. Configure a new worker group called `storage` for the cluster
++
+[source,bash]
+----
+pushd "inventory/classes/${TENANT_ID}"
+yq -i e '.parameters.openshift4_terraform.terraform_variables.additional_worker_groups=
+  {
+    "storage": {
+      "count": 3,
+      "flavor": "plus-32"
+    }
+  }' "${CLUSTER_ID}.yml" 
+
+git commit -am"Configure storage cluster nodes for ${CLUSTER_ID}"
+git push origin master
+popd
+----
+
+. Compile and push the cluster catalog
++
+[source,bash]
+----
+commodore catalog compile "${CLUSTER_ID}" --push -i
+----
+
+. Navigate to the cluster catalog repository and trigger the Terraform apply step if the plan output looks ok.
```

Comment:
I actually didn't know that was setup.

Reply:
Well, you have to enable it manually, and I'm not sure whether this is actually documented somewhere...